source/parser.hera

```diff
@@ -4069,7 +4072,18 @@ ImplicitExportSpecifier
 Declaration
   HoistableDeclaration
   ClassDeclaration
-  LexicalDeclaration
+  LexicalDeclaration:d ->
+    if (d.thisAssignments?.length)
+      return {
+        ...d,
+        children: [...d.children, ...d.splices, ";", ...d.thisAssignments]
```

Comment:
Should we just have a general mechanism for postfixing statements here? Or is there a need to have splices and thisAssignments separated?

Reply:
The splices become part of the declaration so I don't think it can be generalized.